libs/ob-sql-parser/src/main/resources/obmysql/sql/OBParser.g4

```diff
@@ -2345,6 +2548,11 @@ relation_factor_in_use_join_hint_list
     | relation_factor_in_use_join_hint_list relation_sep_option LeftParen relation_factor_in_hint_list RightParen
     ;
 
+coalesce_strategy_list
```

Comment:
this expression is never used, we may delete it

Reply:
done